taskcluster/ci/release-notify-ship/kind.yml

```diff
@@ -0,0 +1,43 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: fenix_taskgraph.loader.multi_dep:loader
+
+transforms:
+    - fenix_taskgraph.transforms.multi_dep:transforms
+    - taskgraph.transforms.remove_upstream_artifacts:transforms
+    - taskgraph.transforms.release_notifications:transforms
+    - taskgraph.transforms.task:transforms
+
+kind-dependencies:
+    - push-apk
+    - github-release
+    - version-bump
+    - mark-as-shipped
+
+primary-dependency: push-apk
+
+group-by: build-type
+
+only-for-build-types:
+    - beta
+    - release
+
+job-template:
+    attributes:
+        shipping_phase: ship
+    name: notify-release-drivers-ship
+    description: Sends email to release-drivers telling release was shipped.
+    worker-type: succeed
+    worker:
+        implementation: succeed
+    notifications:
+        subject: "{config[params][project]} {release_config[version]} build{release_config[build_number]} has been publicly shipped"
```

Comment:
@rvandermeulen @sv-ohorvath Same question about this wording 


<img width="536" alt="image" src="https://user-images.githubusercontent.com/5907366/181763764-d1199cea-ae91-43d8-a44c-ec73cbdfa34d.png">

Reply:
I'm less crazy about the wording on this one. I think for now, we should say something along the lines of "has been tagged in GitHub" or something. I guess longer-term, if we want to to have ship-it handle the actual Play Store promotions & rollout changes, we can update the wording at that time to reflect what's going on.

Reply:
Same, sounds good

Reply:
Sounds good. I changed the message into: "has been tagged in GitHub"